hw/ip/pwrmgr/doc/checklist.md

```diff
@@ -221,11 +221,11 @@ Review        | [V3_CHECKLIST_SCOPED][]                 | Done        |
 
  Type         | Item                                    | Resolution  | Note/Collaterals
 --------------|-----------------------------------------|-------------|------------------
-Documentation | [SEC_CM_TESTPLAN_COMPLETED][]           | Not Started |
-Tests         | [FPV_SEC_CM_VERIFIED][]                 | Not Started |
-Tests         | [SIM_SEC_CM_VERIFIED][]                 | Not Started |
-Coverage      | [SIM_COVERAGE_REVIEWED][]               | Not Started |
-Review        | [SEC_CM_DV_REVIEWED][]                  | Not Started |
+Documentation | [SEC_CM_TESTPLAN_COMPLETED][]           | Done        |
+Tests         | [FPV_SEC_CM_VERIFIED][]                 | NA          |
```

Comment:
why is this N/A? I think we have pwrmgr fpv right?

Reply:
https://reports.opentitan.org/hw/top_earlgrey/formal/summary/latest/report.html
pwrmgr sec_cm is 100%, i think you can mark it as done

Reply:
You are right. I will update.